### PR TITLE
Style to avoid issues with events

### PR DESCRIPTION
**Pull request by pedrohenriquerls**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2661,6 +2661,23 @@ Other Style Guides
     $sidebar.find('ul').hide();
     ```
 
+  <a name="jquery--attach-events"></a><a name="25.5"></a>
+  - [25.5](#jquery--attach-events) Avoid mess up with the events queue.
```
Comment (ljharb) [inline README.md]: can you elaborate on this a bit more? Is the advice specifically to ignore "off"?

File: README.md [hunk 2]
```diff
@@ -2661,6 +2661,23 @@ Other Style Guides
...
+
+    ```javascript
+    // bad
+    $('.sidebar').off('click').on('click', (e) => {
+     console.log('dont do this again!!');
+    });
+
+    // good
+    $('.sidebar').on('click', (e) => {
+     e.stopPropagation();
```
Comment (ljharb) [inline README.md]: generally speaking, stopping propagation is an impolite thing to the rest of the DOM, and should not be done lightly. Can you elaborate on this a bit?

File: README.md [hunk 3]
```diff
@@ -2661,6 +2661,23 @@ Other Style Guides
...
+     console.log('well done dude!! :)');
```
Comment (ljharb) [inline README.md]: let's please avoid potentially gendered words like "dude"

Comment (pedrohenriquerls) [inline README.md]: ok, I will fix this

File: README.md (hunk 1 above)
Comment (pedrohenriquerls) [inline README.md]: yes, the idea is avoid do this on event attach, I will improve the text

File: README.md (hunk 2 above)
Comment (pedrohenriquerls) [inline README.md]: ok, I will do this thanks for the feedback :+1:

**Comment by pedrohenriquerls** [comment]
@ljharb I improve the explanation about the new style, like you requested :)
[changes](https://github.com/airbnb/javascript/pull/903/files)

**Comment by ljharb** [comment]
Thanks. I'm not sure this is a pattern that happens frequently enough to need commenting on. Certainly calling `off` with no arguments is a destructive action, but it both might be intended, and, I don't see it commonly occurring in jQuery codebases, now or throughout my career. In addition, Airbnb now uses React, so we don't tend to need jQuery for event handling any more, so this is even less likely to come up in the future.

Thanks for the contribution! I'm going to close this for now.

**Pull request closed without merging**